targets/mbed/Makefile.mbed

```diff
@@ -0,0 +1,101 @@
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# use TAB-8
+
+CURDIR          = `pwd`
```

Comment:
I'm just realizing that this assignment is here.
- Does make not set it by default? (From make manual: "when GNU make starts (after it has processed any -C options) it sets the variable CURDIR to the pathname of the current working directory")
- It is not referenced anywhere.

Just remove it?